pkg/controller/volume/attachdetach/statusupdater/node_status_updater.go

```diff
@@ -95,10 +95,10 @@ func (nsu *nodeStatusUpdater) processNodeVolumes(nodeName types.NodeName, attach
 			err)
```

Comment:
Again, don't we want to set update not needed here, so we don't retry when the node doesn't exist?

Reply:
I am not sure what 's the behavior if node with the same name is added back?